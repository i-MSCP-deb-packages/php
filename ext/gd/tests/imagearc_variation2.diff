001+ 030018165cad7a4ca0b6a262c3794cca
001- 463b4aea9d9acfab30016ee92613c779