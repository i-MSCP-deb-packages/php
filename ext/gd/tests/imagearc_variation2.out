030018165cad7a4ca0b6a262c3794cca